### ISSUE TITLE
Feature request: using in wiki filetype too

### ISSUE THREAD

**Issue by alexandreroberts** [sampsyo/autolink.vim#6]
I would like to be able to use the `<leader>am` feature to create reference links below a paragraph while in `.wiki` buffers in addition to `.md` buffers. The syntax for these links is the same as in Markdown, so I think that all that needs to be changed is that this functionality is activated for wiki buffers.

**Comment by sampsyo** [comment]
Good idea! Maybe it's as easy as adding the appropriate file types to these checks:
https://github.com/sampsyo/autolink.vim/blob/a1c4366337117ed3ffecef9d5994187502370603/autoload/autolink.vim#L163

Any chance you could give that a shot, and open a small PR if it works for you?

**Comment by alexandreroberts** [comment]
This is where I must confess that I am not a programmer and have never actually made a pull request on Github before. I am happy to try it but fear I might mess something up. I think what I would do is `git clone` this repository onto my computer, add a new branch, make the proposed edit, commit it, then do `git push`. Is that right?

As for the code itself, would it be as simple as changing the line you have linked to (line 163) to read as follows?
```vim
 if (&filetype == "markdown" || &filetype == "mkd" || &filetype == "wiki") 
```
I'm guessing it would, since I am proposing that this plugin treat wiki files exactly like markdown files, but maybe I am missing something?

Sorry for my ignorance. This is what happens when a humanities researcher tries to use vim...

**Comment by alexandreroberts** [comment]
OK, I think I figured out how to do the cloning part and even managed to test it -- and it works! But now when I write `git push` I get this message:
```
remote: Permission to sampsyo/autolink.vim.git denied to alexandreroberts.
fatal: unable to access 'https://github.com/sampsyo/autolink.vim/': The requested URL returned error: 403
```
Is there some way to propose the change without needing permission?

**Comment by alexandreroberts** [comment]
OK, I think I have found the answer [here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork): I am supposed to fork the repository first. I'll try that now.

**Comment by alexandreroberts** [comment]
There, I think I have [done it](https://github.com/sampsyo/autolink.vim/pull/7#issue-1640641055). Hopefully this is what I was supposed to do --- apologies if something is off.
